exercises/dict/dict1.cairo

```diff
@@ -0,0 +1,29 @@
+// dict1.cairo
+// The Felt252Dict maps a felt252 to a value of the specified type.
+// In this exercise, you will map a `u32` to a `felt252`.
+
+// Your task is to create an `Felt252Dict` which holds three elements of type `u32`.
+// The first element shoud map the key 'A' to the value 1, the second key 'B' to the value 2 
+// and the third shoud map 'bob' to the value 3.
+// Make me compile and pass the test!
+// Execute `starklings hint dict1` or use the `hint` watch subcommand for a hint.
+
+// I AM NOT DONE
+
+
+fn create_dictionary() -> Felt252Dict<u32> {
+    let mut a: Felt252Dict<u32> = Default::default();
+    //TODO
+
+}
+
+
+// Don't change anything in the test
+#[test]
```

Comment:
You need to add a line below with 

```rust
#[available_gas(200000]
```

Otherwise we might run into some issues